oneview-ansible.md

```diff
@@ -3526,20 +3526,6 @@ Retrieve facts about one or more of the OneView Enclosure Groups.
 
 - debug: var=enclosure_groups
 
-- name: Gather facts about an Enclosure Group by name with configuration script
-  oneview_enclosure_group_facts:
-    hostname: 172.16.101.48
-    username: administrator
-    password: my_password
-    api_version: 1200
-    name: "Test Enclosure Group Facts"
-    options:
-      - configuration_script
-    delegate_to: localhost
-
-- debug: var=enclosure_groups
-- debug: var=enclosure_group_script
-
 ```
```

Comment:
Instead of deleting all the playbooks and library code related to c7000, just leave the code as it is and add "ignore_errors" tag in playbook. So that it can be useful while running on c7000. Similar things are done in logical enclosures. Just pasted it for your reference.

    # This operation is only available on HPE c7000
    - name: Create a Logical Enclosure (available only on HPE c7000)
      oneview_logical_enclosure:
        config: "{{ config }}"
        state: present
        data:
            enclosureUris: "{{ enclosure_uri }}"
            enclosureGroupUri: "{{ enclosure_group_uri }}"
            name: "{{ logical_enclosure_name }}"
      ignore_errors: true
      delegate_to: localhost

Reply:
This functionality is not supported from apiVersion 500 and if we see in endpoints its evident that we are providing support from 500, so I think keeping it doesn't make much sense.

Reply:
ok